cantaloupe.properties.prod

```diff
@@ -126,14 +126,14 @@ endpoint.iiif.2.restrict_to_sizes = false
 # Enables the Control Panel, at /admin.
 endpoint.admin.enabled = true
 endpoint.admin.username = admin
-endpoint.admin.secret = admin
```

Comment:
You mentioned there's more work to be done here. Are we at all worried about these changes accidentally getting released? Would it be better not to enable at all in the prod config until the solution is working?

Reply:
This is a great point.

I think what we should do is make this PR so that it won't accidentally enable production without a secure password. So that means setting `endpoint.admin.enabled = false` here. Then in production, we use environment variable overrides to set that to true, as well as set the password.

PR updated!